src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
I don't see any reason to use it here

Reply:
I remove it. Thanks

Reply:
Actually it is mine) Forgot to remove it before merging code with @lastvova